.circleci/config.yml

```diff
@@ -9,24 +9,36 @@ version: 2.0
 jobs:
   build-doc:
     docker:
-      - image: circleci/python:3.7.3-stretch
+      - image: cimg/python:3.8.12
 
     working_directory: ~/tmp-fairlearn
     steps:
       # check out PR branch
       - checkout
 
+      - run:
+          name: Show Python version
+          command: python --version
+
+      - run:
+          name: Update apt
+          command: sudo apt update
```

Comment:
why do we need this?

Reply:
I couldn't get pandoc to install otherwise. It's possible that it's not the thing which did it, though. I can check.

Reply:
Just confirmed; without it installing `pandoc` fails:

```bash
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package pandoc
E: Unable to locate package pandoc-citeproc

Exited with code exit status 100
CircleCI received exit code 100
```